CONTRIBUTING.md

```diff
@@ -27,6 +27,81 @@ KeystoneJS adheres to the [Contributor Covenant Code of Conduct](/CODE-OF-CONDUC
 
 KeystoneJS follows the [Thinkmill Monorepo Style Guide](https://github.com/Thinkmill/monorepo). For more information on the reasoning behind using certain tooling, please refer to it.
 
+### Installation
```

Comment:
We should be verbose that these steps are for linking a local copy (or fork) of the `keystone-6/*` monorepo for local development in an external project.

If you are contributing to `keystone-6/*` generally, you don't need to follow these steps beyond `git clone https://github.com/keystonejs/keystone.git` and `yarn` in the monorepo root.

Reply:
Well I missed `packages/core/bin/cli.js dev` to run keystone from the monrepo as `keystone dev` return `keystone command not found`. Maybe adding a script line "dev" in package.json would help. You also need to create a config file at the monorepo root to run it. Or I missed something.

Actually I first try to run a fork as an Embed keystone in next with `withKeystone`, this is why I messed up (and this is why I did first a commit to prevent using env.PORT as it is used by next).

So possible changes :
- For clarity : add a script `"dev": "packages/core/bin/cli.js dev"` command in package.json at monorepo root. Add information about the config file and `yarn dev` after `yarn install` in CONTRIBUTING.md
- complete the CONTRIBUTING.md example as you suggest for those who want to work with an embed keystone in next or an external project.
- create an issue about env.PORT priority in `packages/core/src/scripts/run/dev.ts` which is used by next and not configurable without .env